src/main/java/main/TessUi.java

```diff
@@ -0,0 +1,124 @@
+package main;
+
+import java.util.Scanner;
+
+/**
+ * Represent the user interface of Tesseract.
+ * @author Fan Jue
+ * @version 0.1.0
+ * @since 0.1.0
+ */
+public class TessUi {
+    // deals with interactions with the user
+    protected static final String INDENT_ONE = "    ";
+    protected static final String INDENT_TWO = "        ";
+    protected static final String BREAKER = "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~";
+    protected Scanner sc;
+
+    TessUi() {
+        sc = new Scanner(System.in);
+    }
+
+    String readCommand() {
+        return sc.nextLine();
+    }
+
+    /**
+     * Display the welcome message to the user.
+     */
+    void sayHi() {
+        System.out.println(BREAKER + "\n"
+                + "Hi fellow! I am Tesseract\n"
+                + "I can bring you to wherever you want in the universe\n"
+                + "How can I help you?\n"
+                + BREAKER);
+    }
+
+    /**
+     * Display the farewell message to the user.
+     */
+    public void sayBye() {
+        System.out.println(BREAKER + "\n"
+                + "Ok I'm gonna travel to another planet now\n"
+                + "Hope to see you again when I'm back :P\n"
+                + BREAKER);
+    }
+
+    void display(String msg) {
+        String res = INDENT_ONE + BREAKER + "\n"
+                + INDENT_ONE + msg + "\n"
+                + INDENT_ONE + BREAKER + "\n";
+        System.out.println(res);
+    }
+
+    /**
+     * Display the list of tasks to the user.
+     */
+    public void listTasks(String msg) {
+        String res = "Look what I have noted down for you~ \n" + msg;
+        display(res);
+    }
+
+    /**
+     * Display the deleted task to the user.
+     */
+    public void deleteTaskRes(String msg, int numOfTask) {
+        String res = "Okies the following task has been removed:\n"
+                + INDENT_TWO + msg + "\n"
+                + INDENT_ONE + "Now you have " + numOfTask + " tasks in the list~\n";
+        display(res);
+    }
+
+    /**
+     * Display the added task to the user.
+     */
```

Comment:
Is there a missing explanation for the input parameter? Could it be the same for the rest of the methods in this file?